File: coredata/ObjC/_CurrentPersonWrapper.m
#import "_CurrentPersonWrapper.h"
#import "CurrentPerson.h"

@interface _CurrentPersonWrapper ()
@end

@implementation _CurrentPersonWrapper {
}
- (instancetype)initWithEntity:(CurrentPerson *)entity {
    self = [super init];
    if (self) {
        _entity = entity;
    }
    return self;
}

- (void)updateEntity:(CurrentPerson *)entity {
}
@end

File: coredata/Swift/_CurrentPersonWrapper.swift
import Foundation

class _CurrentPersonWrapper: NSObject {
    let _entity: CurrentPerson?
    func entity() -> CurrentPerson? {
        return _entity
    }

    init(currentPerson: CurrentPerson?) {
        _entity = currentPerson
        super.init()
        if let e = currentPerson {
        }
    }
    func updateCurrentPerson(currentPerson: CurrentPerson) {
    }
}
